ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,81 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Create OpenStack namespace
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: namespace
+        params: &install_yamls_out
```

Comment:
you will need to add MICROSHIFT:1 to the parms now
i have updated my patch to be conditinal on that new parmater
this can also be done via the invokeation of the openstack target

Reply:
Done